### PR TITLE
Pin Page

### PR DESCRIPTION
**Pull request by MariaS12**

### CODE REVIEW

File: src/Pages/Auth/Pin/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  return (
+    <div>
+      <h3>{`${t("int.pin_code")}`}</h3>
+      <form>
+        <label>
+          {`${t("int.pin")}`}
+          <Input
+            type="password"
+            onChange={(value) => setFormData({ ...formData, email: value })}
```
Comment (nikoskleidis) [inline src/Pages/Auth/Pin/index.js:24]:
```suggestion
            onChange={(value) => setFormData({ ...formData, pin: value })}
```

File: src/App.js [hunk 2]
```diff
@@ -32,6 +33,14 @@ function App() {
                 />
               </LayoutAuth>
             </Route>
+            <Route path="/pin">
+              <LayoutAuth>
+                <Pin
+                  loggedStatus={loggedStatus}
+                  setLoggedStatus={setLoggedStatus}
```
Comment (nikoskleidis) [inline src/App.js:40]: I don't think loggedStatus and setLoggedStatus are needed here

**Comment by nikoskleidis** [comment]
Also can you revert the changes to package.lock. I don't know why those changes where inserted in the first place

**Comment by MariaS12** [comment]
> Also can you revert the changes to package.lock. I don't know why those changes where inserted in the first place




> Also can you revert the changes to package.lock. I don't know why those changes where inserted in the first place

Neither do I. I don't know what I should do

**Pull request merged**